images/devtools-golang-v1beta1/tests/prototype/main.go

```diff
@@ -10,4 +11,22 @@ import (
 func main() {
 	fmt.Println("2 + 2 =", sum.Sum(2, 2))
 	fmt.Println("os.Args =", os.Args)
+
+	err := badError()
+	if err != nil {
+		panic(err)
+	}
+
+	err = goodError()
+	if err != nil {
+		panic(err)
+	}
+}
+
+func badError() error {
+	return errors.New("This is an error") //nolint:staticcheck
+}
+
+func goodError() error {
+	return errors.New("this is an error")
 }
```

Comment:
Enforcing x reflects the recommendations from [Go Wiki: Go Code Review Comments](https://go.dev/wiki/CodeReviewComments)

> [Error Strings](https://go.dev/wiki/CodeReviewComments#error-strings)
Error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation, since they are usually printed following other context. That is, use fmt.Errorf("something bad") not fmt.Errorf("Something bad"), so that log.Printf("Reading %s: %v", filename, err) formats without a spurious capital letter mid-message. This does not apply to logging, which is implicitly line-oriented and not combined inside other messages.

We can decide to enforce this in a separate change for propper discussion.